inkcut/device/plugin.py

```diff
@@ -824,6 +824,10 @@ def submit(self, job, test=False):
                             yield defer.maybeDeferred(connection.write,
                                                       config.commands_after)
 
+                        # fix plotter skipping the end of job by sending a lot of 'do nothing'
```

Comment:
This is not something most users will need.

Reply:
"This is not something most users will need."
as any option existing on a single particular model !

I have checked with wireshark, the data is sent to the plotter, no errors from the PC side 'a priori'.
Sadly, I know no other person using the controller board to see if they have the bug too.
It's a filler  sequence of NOP no operation, not needed in 99,9% machines, yes, but I can assure you it fixes end of job not interpreted on my plotter.

Yes I can fill the 'after job sequence' with a thousand of NOP.

But in any case, it is way more easy to deactivate an annoying function than having to remember (or even think) to add it when needed.

Maybe make this another option (turned by default) is a good compromise.


(you can fix areabase independently of this if you want)

Reply:
Maybe a good compromise would be to add the ability to run a script before and after a job?

Reply:
that does not solve the problem of the guy when the guy is a user and not  'a researcher' that will find a possible solution here.

Try it with your cutter and see it does not add time nor problem to the job (except a  longer debug).

Again, it is easier to enable/disable something existing (you explicitly read it) than guessing you have to add something here for that particular problem when you have it...

I have added some detail here #182 . Those cutter are sold $200 $250, probably a lot of them, sure 0 updates support to expect from the manufacturer.